### PR TITLE
teleconsole 0.3.1 (new formula)

### PR DESCRIPTION
**Pull request by DoomHammer**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/teleconsole.rb
Comment (bfontaine) [inline Formula/teleconsole.rb]: Why is this necessary? Is upstream aware of the issue?

File: Formula/teleconsole.rb
Comment (bfontaine) [inline Formula/teleconsole.rb]: You can probably avoid this `bin.install` by using `system "make", "OUT=#{bin}/teleconsole"`.

File: Formula/teleconsole.rb
Comment (DoomHammer) [inline Formula/teleconsole.rb]: Not sure if they know. Found the solution here: https://aur.archlinux.org/packages/teleconsole/

Comment (MikeMcQuaid) [inline Formula/teleconsole.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/teleconsole.rb
Comment (DoomHammer) [inline Formula/teleconsole.rb]: Reported: https://github.com/gravitational/teleconsole/pull/8

Comment (MikeMcQuaid) [inline Formula/teleconsole.rb]: Will hold off until we seem some movement and, ideally, a merge.

**Comment by DoomHammer** [comment]
Merged upstream. Should we wait for a new release now?

**Comment by MikeMcQuaid** [comment]
@DoomHammer I'm cool with merging almost as-is once there's a comment added above the `inreplace` pointing to the PR and explaining why it's needed. Ideally it may be worth using a `patch do` with the PR's `.patch` URL instead, too (with the same comment).

**Comment by DoomHammer** [comment]
Updated with a GitHub patch.

File: Formula/teleconsole.rb
Comment (MikeMcQuaid) [inline Formula/teleconsole.rb]: Don't need `:p1`, it's the default.

File: Formula/teleconsole.rb
Comment (MikeMcQuaid) [inline Formula/teleconsole.rb]: Favour `.patch` and the PR `.patch` because it's easier to strip the `.patch` and see more context. Additionally, needs a comment above noting what the patch is needed for and that it was merged upstream.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/teleconsole.rb
Comment (DoomHammer) [inline Formula/teleconsole.rb]: I don't quite get the first sentence. Should I put the patch verbose in Formula and link to the upstream commit?

Comment (MikeMcQuaid) [inline Formula/teleconsole.rb]: Sorry, use https://github.com/gravitational/teleconsole/pull/8.patch as the URL instead.

Comment (dunn) [inline Formula/teleconsole.rb]: Ping on this!

Comment (DoomHammer) [inline Formula/teleconsole.rb]: @MikeMcQuaid ?

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
@BrewTestBot test this please

**Pull request closed without merging**
